### ISSUE TITLE
自己训练的模型转onnx失败

### ISSUE THREAD

**Issue by 205418367** [JDAI-CV/FaceX-Zoo#105]
识别模型：ReXNetV1
转换脚本：
model_path="out_dir/Epoch_249.pt"
onnx_path="models/ReXNetV1.onnx"

class OnnxConvertor:
    def __init__(self):
        pass
    def convert(self, model, output_path):
        img = np.random.randint(0, 255, size=(112,112,3), dtype=np.uint8)
        img = img[:,:,::-1].astype(np.float32)
        img = (img.transpose((2, 0, 1)) - 127.5) * 0.0078125
        img = torch.from_numpy(img).unsqueeze(0).float()
        torch.onnx.export(
            model,
            img,
            output_path,
            keep_initializers_as_inputs=False,
            verbose=False,
            opset_version=12
        )
        # batch inference.
        onnx_model = onnx.load(output_path)
        graph = onnx_model.graph
        graph.input[0].type.tensor_type.shape.dim[0].dim_param = 'None'
        onnx.save(onnx_model, output_path)

if __name__ == '__main__':
    model = ReXNetV1(input_ch=16, final_ch=180, width_mult=1.0, depth_mult=1.0,
                 use_se=0, se_ratio=12, out_h=7, out_w=7, feat_dim=512, dropout_ratio=0.2)
    #model.load_state_dict(torch.load(model_path)["state_dict"])
    model_dict = model.state_dict()
    pretrained_dict = torch.load(model_path)['state_dict']
    new_pretrained_dict = {}
    for k in model_dict:
        if k.startswith("features."):
            print(k)
            new_pretrained_dict[k] = pretrained_dict[k]
    model_dict.update(new_pretrained_dict)
    model.load_state_dict(model_dict)
    model.eval()
    #dummy_input = torch.randn(1, 3, 112, 112)
    #torch.onnx.export(model, dummy_input, onnx_path, opset_version=11)
    convertor = OnnxConvertor()
    convertor.convert(model, onnx_path)

**Comment by Phxmark** [comment]
如何解决的 能把脚本分享一下吗
